### PR TITLE
[MODFEE-36] Update RMB to v29.3.1

### PR DESCRIPTION
**Pull request by OleksandrVidinieiev**
Update RMB to v29.3.1, update Vert.x to v3.8.5

Resolves [MODFEE-36](https://issues.folio.org/browse/MODFEE-36).

### CODE REVIEW

File: src/test/java/org/folio/rest/impl/LostItemFeePoliciesAPITest.java [hunk 1]
```diff
@@ -41,207 +43,223 @@
 
 @RunWith(VertxUnitRunner.class)
 public class LostItemFeePoliciesAPITest {
```
Comment (alexanderkurash) [inline src/test/java/org/folio/rest/impl/LostItemFeePoliciesAPITest.java]: Indentation is not the only change in this class, is that correct?

File: src/main/java/org/folio/rest/utils/ErrorHelper.java [hunk 2]
```diff
@@ -9,23 +9,36 @@
...
+      .filter(msg -> msg.contains(String.format(DUPLICATE_NAME_MSG_TEMPLATE, tableName)))
       .isPresent();
   }
 
   public static Errors createErrors(Error... errors){
     return new Errors()
       .withErrors(Arrays.asList(errors));
   }
+
+  public static Errors createSingleError(String message, String code) {
```
[the remark this answers is not in the document]
Comment (OleksandrVidinieiev) [reply src/main/java/org/folio/rest/utils/ErrorHelper.java]: Done.

File: src/test/java/org/folio/rest/impl/LostItemFeePoliciesAPITest.java (hunk 1 above)
Comment (OleksandrVidinieiev) [reply src/test/java/org/folio/rest/impl/LostItemFeePoliciesAPITest.java]: Yes. I brought back the original indentation.

Comment (alexanderkurash) [reply src/test/java/org/folio/rest/impl/LostItemFeePoliciesAPITest.java]: Thanks, it makes it much easier to review. We'll fix indentation some time later with a separate PR.

**Pull request merged**